packages/l10n/scripts/missing-countries.js

```diff
@@ -0,0 +1,82 @@
+const missingCountries = {
+  en: {
+    AN: 'Netherlands Antilles',
+    BU: 'Burma',
+    CS: 'Serbia and Montenegro',
+    FX: 'France, Metropolitan',
+    NT: 'Neutral Zone',
+    SF: 'Finland',
+    SU: 'USSR',
+    TP: 'East Timor',
+    UK: 'United Kingdom',
+    YU: 'Yugoslavia',
+    ZR: 'Zaire',
+  },
+  de: {
+    AN: 'Niederländische Antillen',
+    BU: 'Birma',
+    CS: 'Serbien und Montenegro',
+    FX: 'Frankreich, Metropolit',
+    NT: 'Neutrale Zone',
+    SF: 'Finnland',
+    SU: 'UdSSR',
+    TP: 'Osttimor',
+    UK: 'Vereinigtes Königreich',
+    YU: 'Jugoslawien',
+    ZR: 'Zaire',
+  },
+  es: {
+    AN: 'Antillas Holandesas',
+    BU: 'Birmania',
+    CS: 'Serbia y Montenegro',
+    FX: 'Francia, Metropolitana',
+    NT: 'Zona neutral',
+    SF: 'Finlandia',
+    SU: 'URSS',
+    TP: 'Timor Oriental',
+    UK: 'Reino Unido',
+    YU: 'Yugoslavia',
+    ZR: 'Zaire',
+  },
+  'fr-FR': {
+    AN: 'Antilles néerlandaises',
+    BU: 'Birmanie',
+    CS: 'Serbie et Monténégro',
+    FX: 'France métropolitaine',
+    NT: 'Zone neutre',
+    SF: 'Finlande',
+    SU: 'URSS',
+    TP: 'Timor oriental',
+    UK: 'Royaume-Uni',
+    YU: 'Yougoslavie',
+    ZR: 'Zaïre',
+  },
+  'zh-CN': {
+    AN: '荷属安的列斯',
+    BU: '缅甸',
+    CS: '塞尔维亚和黑山',
+    FX: '法国，大都会',
+    NT: '中立区',
+    SF: '芬兰',
+    SU: '苏联',
+    TP: '东帝汶',
+    UK: '英国',
+    YU: '南斯拉夫',
+    ZR: '扎伊尔',
+  },
+  ja: {
```

Comment:
If I recall correctly, removed Japanese translation support, so this object is probably not necessary anymore